### PR TITLE
changed theme, removed unnecessary sections, fixed hyperlinks of navbar

### PR DESCRIPTION
**Pull request by Garvit-tyagi**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -210,153 +209,10 @@ <h4><a href="">Case Solving Competitions</a></h4>
...
-            <p>Hard Workers</p>
-          </div>
-
-        </div>
-
-      </div>
-    </section><!-- End Counts Section -->
+    
 
     <!-- ======= Portfolio Section ======= -->
```
Comment (9401adarsh) [inline index.html:214]: Just Comment it as Members section to get a better clarity

**Review by 9401adarsh — APPROVED** [review]

**Review by sidvenu — CHANGES_REQUESTED** [review]
1. Remove Inductions and Get Started from navbar
2. Add ProfNITT's official logo to navbar and footer instead of plain text
3. Add ProfNITT's circle logo to Title bar (favicon)
4. Change the background image, it looks pixelated.

File: index.html [hunk 2]
```diff
@@ -68,11 +67,11 @@ <h1 class="logo"><a href="#">ProfNITT</a></h1>
                 </ul>
               </li>
               <li><a href="#contact">Contact</a></li>
-              <li><a href="#inductions">Inductions</a></li>
+              <!-- <li><a href="#inductions">Inductions</a></li> -->
```
Comment (sidvenu) [inline index.html]: Remove

File: index.html [hunk 3]
```diff
@@ -43,22 +43,21 @@
 
       <div class="row justify-content-center">
         <div class="col-xl-9 d-flex align-items-center justify-content-between">
-          <h1 class="logo"><a href="#">ProfNITT</a></h1>
+          <!-- <h1 class="logo"><a href="#">ProfNITT</a></h1> -->
```
Comment (sidvenu) [inline index.html]: Remove

File: index.html [hunk 4]
```diff
@@ -628,7 +431,8 @@ <h4>Call:</h4>
   <!-- ======= Footer ======= -->
   <footer id="footer">
     <div class="container">
-      <h3>ProfNITT</h3>
+      <!-- <h3>ProfNITT</h3> -->
```
Comment (sidvenu) [inline index.html]: Remove

**Review by sidvenu — CHANGES_REQUESTED** [review]

**Review by 9401adarsh — APPROVED** [review]

**Review by sidvenu — APPROVED** [review]

**Pull request merged**